Signal/src/ViewControllers/ConversationView/ConversationViewController.m

```diff
@@ -3505,13 +3505,6 @@ - (void)navigationTitleLongPressed:(UIGestureRecognizer *)gestureRecognizer
 
 #pragma mark - ConversationInputTextViewDelegate
 
-- (void)inputTextViewDidBecomeFirstResponder
-{
-    OWSAssert([NSThread isMainThread]);
-
-    [self scrollToBottomAnimated:YES];
```

Comment:
Don't scroll to bottom when we start editing text messages - scroll to bottom when we send.